cmd/sqlflowserver/main_test.go

```diff
@@ -1038,13 +1066,24 @@ INTO sqlflow_models.my_xgb_regression_model;
 	}
 	// call ParseRow only to wait train finish
 	ParseRow(stream)
+}
+
+func CasePredictXGBoostRegression(t *testing.T) {
+	a := assert.New(t)
+	conn, err := createRPCConn()
+	a.NoError(err)
+	defer conn.Close()
+	cli := pb.NewSQLFlowClient(conn)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+	defer cancel()
 
 	predSQL := fmt.Sprintf(`SELECT *
 FROM housing.test
 PREDICT housing.xgb_predict.target
 USING sqlflow_models.my_xgb_regression_model;`)
```

Comment:
Add some indents?

Reply:
I'm not sure which style of the SQL string is better. How about following the former one? 
In fact, I think this style is pretty good.